app/Http/Transformers/ReportbackItemTransformer.php

```diff
@@ -19,6 +19,7 @@ public function transform(ReportbackItem $reportbackItem)
             'id' => (string) $reportbackItem->id,
             'reportback_id' => $reportbackItem->reportback_id,
             'file_id' => $reportbackItem->file_id,
+            'file_url' => $reportbackItem->file_url,
```

Comment:
Thoughts on maybe changing this to be a media item and inside it it contains the id, url, etc... kinda like it does now in the RB items resource endpoint? 
![image](https://cloud.githubusercontent.com/assets/105849/18053974/f8a4fe3e-6dcf-11e6-935f-d5ec5cc9e872.png)

Having objects that contain further properties helps avoid having to have a long list of properties with special prefixes `file_something`... 

We could keep it named file if that ends up being preferred, but something more like:
```
'media' => [
  'id' => $reportbackItem->file_id,
  'url' => $reportbackItem->file_url,
]
```

thoughts?